_inc/client/plans/single-product-backup.jsx

```diff
@@ -61,24 +63,54 @@ export function SingleProductBackup( { products, upgradeLinks } ) {
 	);
 }
 
-function SingleProductBackupHeader( { currencySymbol, backupPlanPrices } ) {
+function SingleProductBackupHeader( { currencySymbol, currencyCode, backupPlanPrices } ) {
 	return (
 		<div className="single-product-backup__header-container">
 			<h3 className="single-product-backup__header-title">{ __( 'Jetpack Backup' ) }</h3>
-			<PlanPriceDisplay currencySymbol={ currencySymbol } backupPlanPrices={ backupPlanPrices } />
+			<PlanPriceDisplay
+				currencySymbol={ currencySymbol }
+				currencyCode={ currencyCode }
+				backupPlanPrices={ backupPlanPrices }
+			/>
 		</div>
 	);
 }
 
-export function PlanPriceDisplay( { backupPlanPrices, currencySymbol } ) {
+function formatCurrencyNumber( number, code ) {
+	const options = arguments.length > 2 && arguments[ 2 ] !== undefined ? arguments[ 2 ] : {};
+	const currencyDefaults = getCurrencyDefaults( code );
+
+	if ( ! currencyDefaults || isNaN( number ) ) {
+		return null;
+	}
+
+	const _currencyDefaults$opt = _objectSpread( {}, currencyDefaults, options ),
+		decimal = _currencyDefaults$opt.decimal,
+		grouping = _currencyDefaults$opt.grouping,
+		precision = _currencyDefaults$opt.precision;
+
+	return numberFormat( Math.abs( number ), {
+		decimals: precision,
+		thousandsSep: grouping,
+		decPoint: decimal,
+	} );
+}
+
+export function PlanPriceDisplay( { backupPlanPrices, currencySymbol, currencyCode } ) {
 	const dailyBackupYearlyPrice = backupPlanPrices.jetpack_backup_daily.yearly;
 	const dailyBackupMonthlyPrice = backupPlanPrices.jetpack_backup_daily.monthly;
 
 	const realtimeBackupYearlyPrice = backupPlanPrices.jetpack_backup_realtime.yearly;
 	const realtimeBackupMonthlyPrice = backupPlanPrices.jetpack_backup_realtime.monthly;
 
-	const fullDailyBackupYearlyCost = dailyBackupMonthlyPrice * 12;
-	const fullRealtimeBackupYearlyCost = realtimeBackupMonthlyPrice * 12;
+	const fullDailyBackupYearlyCost = formatCurrencyNumber(
```

Comment:
I guess we could use the `formatCurrency` function directly from '@automattic/format-currency' here. We could tell it to drop the decimals and show the currency symbol only for the first price. Eventually we could land with something like this:

```js
<div className="discounted-price__price">
	{ __( '%(lowPrice)s-%(highPrice)s', {
		args: {
			lowPrice: formatCurrency(
				dailyBackupMonthlyPrice * 12,
				currencyCode,
				{
					precision: 0,
				}
			),
			highPrice: formatCurrency(
				realtimeBackupMonthlyPrice * 12,
				currencyCode,
				{
					precision: 0,
					symbol: '',
				}
			),
		},
		comment:
			"Describes how much a plan will cost per year. %(currencySymbol) is the currency symbol of the user's locale (e.g. $). %(planPrice) is the cost of a plan (e.g. 20).",
	} ) }
</div>
```

The same principle would apply to the non-discounted price.